### PR TITLE
Adding Reactive java WebApp, with Eclipse Vert.x

### LINKED ISSUE #448: Add sample for Eclipse Vert.x
## Actual Behavior

Adding in Eclipse Vert.x sample, to show Rx-Java in action, for a reactive web-application with the Eclipse Vert.x runtime

### PR DESCRIPTION
**Pull request by matzew**
Fixes #448

## Proposed Changes

* adding hello-world w/ Rx-JAva and Eclipse Vert.x and documentation for running that as a Knative serving function

### CODE REVIEW

**Comment by sjwoodman** [comment]
👍

**Comment by matzew** [comment]
@labadav or @vdemeester any concerns regarding adding the `/ok-to-test` label to this PR?

**Comment by grantr** [comment]
@markfisher @scothis I think you're both more qualified to review this than me. :smile: 

/cc @markfisher @scothis

**Comment by matzew** [comment]
@evankanderson you mind taking a look ?

File: serving/samples/helloworld-vertx/README.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            requestFlowable.subscribe(httpServerRequest -> {
+
+                String target = System.getenv("TARGET");
+                if (target == null) {
+                    target = "NOT SPECIFIED";
+                }
+
+                httpServerRequest.response().setChunked(true)
+                        .putHeader("content-type", "text/plain")
+                        .setStatusCode(201) // accepted
```
Comment (markfisher) [inline serving/samples/helloworld-vertx/README.md]: the actual sample Java file included in this same PR responds with `200` (and that's probably more appropriate in this case?)

**Comment by matzew** [comment]
I did address the suggested fix.  Anything else preventing this from being merged ?

**Comment by matzew** [comment]
@RichieEscarez any look here ?

**Pull request merged**